test/balloon/a11y-spec.js

```diff
@@ -3,44 +3,68 @@ import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import Balloon from '../../src/balloon/index';
 import '../../src/balloon/style';
-import { afterEach as a11yAfterEach, test, mountReact } from '../util/a11y/validate';
+import {
+    unmount,
+    test,
+    testReact,
+    createContainer,
+} from '../util/a11y/validate';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-const wrapperClassName = 'js-a11y-test';
-const popupProps = { wrapperClassName };
+const portalContainerId = 'a11y-portal-id';
+let portalContainer;
 
 /* eslint-disable no-undef, react/jsx-filename-extension */
 describe('Balloon A11y', () => {
     let wrapper;
 
     afterEach(() => {
         if (wrapper) {
-            // wrapper.unmount();
+            wrapper.unmount();
             wrapper = null;
         }
-        a11yAfterEach();
+
+        if (portalContainer) {
+            portalContainer.remove();
+        }
+        unmount();
     });
 
     // TODO: fix `button-name` violation for close button
     it.skip('should not have any violations', async () => {
-        wrapper = await mountReact(<Balloon visible popupProps={popupProps}>
+        portalContainer = createContainer(portalContainerId);
+        wrapper = await testReact(
+            <Balloon visible popupContainer={portalContainerId}>
                 I am balloon content
-        </Balloon>);
-        return test(`.${wrapperClassName}`);
+            </Balloon>
+        );
+        return test(portalContainer);
     });
 
     it('should not have any violations when not closable', async () => {
-        wrapper = await mountReact(<Balloon visible closable={false} popupProps={popupProps}>
+        portalContainer = createContainer(portalContainerId);
+        wrapper = await testReact(
+            <Balloon
+                visible
+                closable={false}
+                popupContainer={portalContainerId}
+            >
                 I am balloon content
-        </Balloon>);
-        return test(`.${wrapperClassName}`);
+            </Balloon>
+        );
+
+        return test(portalContainer);
     });
 
     it('should not have any violations when Tooltip', async () => {
-        wrapper = await mountReact(<Balloon.Tooltip visible popupProps={popupProps}>
+        portalContainer = createContainer(portalContainerId);
+
+        wrapper = await testReact(
```

Comment:
Here is the major change. Using `testReact` to test the mounting point, where we use `test` below to just test the portal.